clang/tools/clang-linker-wrapper/SYCLOffloadWrapper.h

```diff
@@ -0,0 +1,57 @@
+//===- SYCLOffloadWrapper.h --r----------------------------------*- C++ -*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_CLANG_TOOLS_CLANG_LINKER_WRAPPER_SYCL_OFFLOAD_WRAPPER_H
+#define LLVM_CLANG_TOOLS_CLANG_LINKER_WRAPPER_SYCL_OFFLOAD_WRAPPER_H
+
+#include "llvm/ADT/SmallVector.h"
+#include "llvm/ADT/StringRef.h"
+#include "llvm/IR/Module.h"
+#include "llvm/Support/PropertySetIO.h"
+
+#include <string>
+#include <utility>
+#include <vector>
+
+// SYCL binary image formats supported.
+enum class SYCLBinaryImageFormat {
+  BIF_None,   // Undetermined Image kind
+  BIF_Native, // Native Image kind
+  BIF_SPIRV,  // SPIR-V
+  BIF_LLVMBC  // LLVM bitcode
+};
+
+struct SYCLImage {
+  std::vector<char> Image;
```

Comment:
Can we rename this as 'Code' to be more specific? Thanks

Reply:
It looks like the "Image" term is already widespread in this area.

For example:
https://github.com/intel/llvm/blob/4cca77c5d372bcce84d728bd84bbdf9576aac20e/clang/tools/clang-linker-wrapper/OffloadWrapper.h#L17-L26